src/API/Baggage/Entry.php

```diff
@@ -6,21 +6,10 @@
 
 final class Entry
 {
-    /** @var mixed */
-    private $value;
-
-    private MetadataInterface $metadata;
-
-    /**
-     * @param mixed $value
-     * @param MetadataInterface $metadata
-     */
     public function __construct(
-        $value,
-        MetadataInterface $metadata
+        private mixed $value,
+        private MetadataInterface $metadata
```

Comment:
```suggestion
        private MetadataInterface $metadata,
```

Should add trailing comma to changed constructors to reduce diff if we add more parameters.

Reply:
added a php-cs-fixer rule to apply this everywhere